File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i = 0; i < nums.size(); i++){
            for(int j = 0; j < nums.size(); j++){
              if(i!=j && nums[i]+nums[j] == target){
                
                vector<int> nums_2{i,j};
                return nums_2;
              }  
            }
        }
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        visited = {}

        for i, n in enumerate(numbers):
            diff = target - n 
            if diff in visited:
                return [visited[diff] + 1, i + 1]
            visited[n] = i